tooling/minimize/tests/pass/zero_size_access.rs

```diff
@@ -0,0 +1,11 @@
+//@revisions: basic tree
+//@[tree]compile-flags: --minimize-tree-borrows
+
+// Check that we allow out-of-range zero-sized accesses.
+
+fn main() {
+    let xraw = &mut 42u8 as *mut u8;
+    let xzst = unsafe { xraw.add(1) as *mut () };
```

Comment:
```suggestion
    let xzst = unsafe { xraw.add(2) as *mut () };
```
To make this really out-of-bounds.

Reply:
Isn't `xraw.add(2)` already UB ? 

https://doc.rust-lang.org/std/primitive.pointer.html#safety-8

Reply:
Ah yes it should be `wrapping_add`.

Reply:
Yeah :D

Reply:
Then for this PR please add a comment like
```
// FIXME(TreeBorrows): This test isn't quite right, the pointer is not actually ouf-of-bounds.
// Use `wrapping_add(2)` once that is possible.
```

Reply:
Ah you have deleted the test. Okay that also works.